app/views/widgets/thermometer/_display.slim

```diff
@@ -0,0 +1,6 @@
+div.progress
+  / get a percentile value of width for the progress bar from count and goal
+  - @width = 100 * options['count'].to_f / options['goal'].to_f
+  div.progress-bar role="progressbar" aria-valuenow=options['count'] aria-valuemin="0" aria-valuemax=options['goal'] style="width: #{@width}%"
```

Comment:
Yes, you're right. I had some serious derp while setting up that widget, it seems.